bedrock/products/templatetags/misc.py

```diff
@@ -54,14 +54,17 @@ def _vpn_get_ga_data(selected_plan):
     return ga_data
 
 
-def _vpn_get_available_plans(country_code, lang):
+def _vpn_get_available_plans(country_code, lang, bundle_monitor_relay):
```

Comment:
I think the tests are failing because they're not being called with `bundle_monitor_relay` if you change the def to default to False they should pass 🤞 

`def _vpn_get_available_plans(country_code, lang, bundle_monitor_relay=False):`